CRM/Utils/System/DrupalBase.php

```diff
@@ -386,16 +386,6 @@ public function getVersion() {
     return defined('VERSION') ? VERSION : 'Unknown';
   }
 
-  /**
-   * @inheritDoc
-   */
-  public function isUserRegistrationPermitted() {
```

Comment:
@agileware-justin Do we want to remove these functions?   I can see a scenario where we break custom code/extensions.   I'd prefer to leave in for now, perhaps set as deprecated.

Reply:
@kcristiano it should be OK, as the system will inherit from the base class, see https://github.com/civicrm/civicrm-core/pull/21182/files#diff-e4db23b0add1458bbca327840556454372551888dd93705c60548765f0adfc6dL449

Reply:
You are correct, my mistake.   I thought the base class was removed.